pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -61,6 +66,20 @@ func (admitter *KubeVirtUpdateAdmitter) Admit(ar *admissionv1.AdmissionReview) *
 	results = append(results, validateCustomizeComponents(newKV.Spec.CustomizeComponents)...)
 	results = append(results, validateCertificates(newKV.Spec.CertificateRotationStrategy.SelfSigned)...)
 
+	if !reflect.DeepEqual(currKV.Spec.Infra, newKV.Spec.Infra) {
+		if newKV.Spec.Infra != nil && newKV.Spec.Infra.NodePlacement != nil {
+			results = append(results,
+				validateInfraPlacement(newKV.Namespace, newKV.Spec.Infra.NodePlacement, admitter.Client)...)
+		}
+	}
+
+	if !reflect.DeepEqual(currKV.Spec.Workloads, newKV.Spec.Workloads) {
+		if newKV.Spec.Workloads != nil && newKV.Spec.Workloads.NodePlacement != nil {
+			results = append(results,
+				validateWorkloadPlacement(newKV.Namespace, newKV.Spec.Workloads.NodePlacement, admitter.Client)...)
+		}
```

Comment:
we need unit test cases for these new validations

Reply:
@davidvossel I was thinking that the e2e will be enough since we don't have mocking for the dry-run.